modules/usage/usage-reporter.php

```diff
@@ -91,4 +91,29 @@ public function get_raw_usage() {
 			'value' => $usage,
 		];
 	}
+
+	/**
+	 * Removes the "elementor_usage_recalc" param from the query string to avoid recalc every refresh.
+	 * When using a redirect header in place of this approach it throws an error because some components have already output some content.
+	 *
+	 * @return string
+	 */
+	private function get_remove_recalc_query_string_script() {
```

Comment:
Try to find a way to print this script as an external file or inside the general Elementor's 'admin.js'.

Reply:
It's not worth the time.

Reply:
@Nevoss Please add a comment with the file path that creates this script - for easy debugging.